### PR TITLE
Mediated transfer integration test

### LINKED ISSUE #1211: Integration test for send transfers
## Description
As a follow up for #348 we need to test a send transfer. To keep the number of tests low, we should test a transfer with the highest complexity (mediated, fees, fees for PFS).

## Acceptance criteria
- 10 Mediated transfers from the LC through a Raiden Python to a Raiden Python client
- Including mediation fees
- Including PFS fees
- Sequentially

## Tasks
- [ ]

### PR DESCRIPTION
**Pull request by nephix**
Resolves #1211

### CODE REVIEW

File: raiden-ts/tests/integration/integration.spec.ts [hunk 1]
```diff
@@ -71,15 +78,55 @@ describe('integration', () => {
...
+      routes = await raiden.findRoutes(getToken(), partner2, 50);
+      expect(routes).toMatchObject([{ fee: bigNumberify(0), path: [partner1, partner2] }]);
+    });
+
+    test('10 consecutive transfers to partner #2', async () => {
+      for (let i = 0; i < 10; i++) {
+        await expect(raiden.getAvailability(partner2)).resolves.toMatchObject({ available: true });
+        await expect(
+          raiden.transfer(getToken(), partner2, 50, { paths: routes }),
+        ).resolves.toMatch('0x');
```
Comment (nephix) [inline raiden-ts/tests/integration/integration.spec.ts:128]: Also did `raiden.waitTransfer()` here but it always results in this matrix error:

> 
> console.warn internal/process/task_queues.js:97
>     Error inviting MatrixError {
>       errcode: 'M_FORBIDDEN',
>       name: 'M_FORBIDDEN',
>       message: '@0xcbc49ec22c93db69c78348c90cd03a323267db86:localhost is already in the room.',
>       data: {
>         errcode: 'M_FORBIDDEN',
>         error: '@0xcbc49ec22c93db69c78348c90cd03a323267db86:localhost is already in the room.'
>       },
>       httpStatus: 403
>     }

and causes transfers to expire and tests to fail.

File: raiden-ts/tests/integration/integration.spec.ts [hunk 2]
```diff
@@ -71,15 +78,55 @@ describe('integration', () => {
...
+
+    test('find routes to partner #2', async () => {
+      await expect(raiden.getAvailability(partner2)).resolves.toMatchObject({ available: true });
+      routes = await raiden.findRoutes(getToken(), partner2, 50);
+      expect(routes).toMatchObject([{ fee: bigNumberify(0), path: [partner1, partner2] }]);
+    });
+
+    test('10 consecutive transfers to partner #2', async () => {
+      for (let i = 0; i < 10; i++) {
+        await expect(raiden.getAvailability(partner2)).resolves.toMatchObject({ available: true });
```
Comment (kelsos) [inline raiden-ts/tests/integration/integration.spec.ts:125]: We don't need to check for availability in the loop though right? we only need to do it the first time

File: raiden-ts/tests/integration/integration.spec.ts [hunk 3]
```diff
@@ -71,15 +78,55 @@ describe('integration', () => {
...
+    });
+
+    test('find routes to partner #2', async () => {
+      await expect(raiden.getAvailability(partner2)).resolves.toMatchObject({ available: true });
+      routes = await raiden.findRoutes(getToken(), partner2, 50);
+      expect(routes).toMatchObject([{ fee: bigNumberify(0), path: [partner1, partner2] }]);
+    });
+
+    test('10 consecutive transfers to partner #2', async () => {
+      for (let i = 0; i < 10; i++) {
```
Comment (kelsos) [inline raiden-ts/tests/integration/integration.spec.ts:124]: any chance to also verify the number of assertions?

File: raiden-ts/tests/integration/integration.spec.ts (hunk 2 above)
Comment (nephix) [reply raiden-ts/tests/integration/integration.spec.ts:125]: Oh yeah that was a left over from the waitTransfer() thing, will fix

**Pull request merged**